config/locales/requests.en.yml

```diff
@@ -86,7 +86,7 @@ en:
       scsb_recall_destination: 'recapproblems@princeton.edu'
     recap_edd:
       email_subject: "Recap Electronic Delivery Confirmation"
-      brief_msg: "Item offsite at ReCAP facility. Request a selection be scanned for you and delivered as a PDF file in 1-2 business days."
+      brief_msg: "Item off-site at ReCAP facility.  Request an article or book chapter be scanned for you and delivered as a PDF file in 1-2 business days.  If you need more than an article or chapter, please request the physical item.  The Library complies with all applicable copyright laws."
```

Comment:
I would expect 1 space after a period. do we have a style guide?

Reply:
Good catch thanks. Princeton Office of Communications has a nice style guide at https://communications.princeton.edu/guides-tools/princeton-editorial-style-guide. We'd probably do well to recommend Library staff who write content for the web (including myself) use that and work with our own coms office and the web steering group to create some additional guidelines that deal with Library location names, how we name our services, other PUL jargon, and acronyms.